tests/test-utils.lisp

```diff
@@ -0,0 +1,48 @@
+(defpackage #:sento.test-utils
+  (:use #:cl)
+  (:import-from #:serapeum
+                #:eval-always)
+  (:import-from #:alexandria
+                #:parse-body)
+  (:export #:parametrized-test))
+(in-package #:sento.test-utils)
+
+
+
+(eval-always
+  (defun generate-test-form (base-test-name parameter-names parameters docstring body-form)
+    (let* ((test-name-str (format nil
+                                  "~A-[~{~A=~S~^ ~}]"
+                                  base-test-name
+                                  (loop :for name :in parameter-names
+                                        :for value :in parameters
+                                        :appending (list name value))))
+           (test-name (intern test-name-str))
+           (bindings (loop :for name :in parameter-names
+                           :for value :in parameters
+                           :collect (list name value))))
+      `(5am:test ,test-name
+         ,docstring
+         (let ,bindings
+           ,@body-form)))))
+
+
+(defmacro parametrized-test (name ((&rest parameter-names) &rest parameter-tuples) &body body)
```

Comment:
Please add a docstring explaining how this is used.

Reply:
Done.